src/v1/controllers/EnrollTotpController.js

```diff
@@ -55,8 +55,11 @@ const EnrollTotpControllerAppDownloadInstructionsView = View.extend({
       appStoreLink = StoreLinks.OKTA[this.model.get('__deviceType__')];
       appIcon = 'okta-verify-download-icon';
     }
+    const appStoreLinkText = appStoreName
```

Comment:
nit: you could set a default `appStoreName`, similar to how above we set a default `appIcon`. Just an idea